scipy/stats/stats.py

```diff
@@ -3692,6 +3694,198 @@ def f_oneway(*args, axis=0):
+    b = [48 * a_i**2 for a_i in a]
+    c = [((a_i * np.log(1 + (t_i ** 2)/v_i))**.5)
+         for a_i, t_i, v_i in zip(a, t_stats, v)]
+
+    # (8) perform a normalizing transformation on t statistic
+    z = [(c_i + ((c_i**3 + 3*c_i)/b_i) -
+          ((4*c_i**7 + 33*c_i**5 + 240*c_i**3 + 855*c_i) /
+           (10*b_i**2 + 8*b_i*c_i**4 + 1000*b_i)))
+         for c_i, b_i in zip(c, b)]
+
+    # (9) calculate statistic
+    A = np.sum(np.square(z))
+
+    # "[the p value is determined from] central chi-square random deviates
+    # with n_i - 1 degrees of freedom". Alexander, Govern (94)
+    p = distributions.chi2.sf(A, len(args) - 1)
+    return AlexanderGovernResult(A, p)
+
+def alexandergovern_alt(*args):
+    """
+    Performs the Alexander Govern test.
+    The Alexander-Govern approximation tests the equality of k independent
+    means in the face of heterogeneity of variance. The test is applied to
+    samples from two or more groups, possibly with differing sizes.
+    Parameters
+    ----------
+    sample1, sample2, ... : array_like
+        The sample measurements for each group.  There must be at least
+        two arguments.
+    Returns
+    -------
+    statistic : float
+        The computed A statistic of the test.
+    pvalue : float
+        The associated p-value from the chi-squared distribution.
+    References
+        ----------
+        ... [1] Alexander, Ralph A., and Diane M. Govern. “A New and Simpler
+                Approximation for ANOVA under Variance Heterogeneity.” Journal
+                of Educational Statistics, vol. 19, no. 2, 1994, pp. 91–101.
+                JSTOR, www.jstor.org/stable/1165140. Accessed 12 Sept. 2020.
+        ... [2] Kingsley Ochuko, T., Abdullah, S., Binti Zain, Z., & Soaad Syed
+                Yahaya, S. (2015). The Modification and Evaluation of the
+                Alexander-Govern Test in Terms of Power. Modern Applied
+                Science, 9(13), 1. https://doi.org/10.5539/mas.v9n13p1
+    Examples
+        --------
+        >>> from scipy.stats import alexandergovern_alt, Kruskal, F_oneway
+        >>> young = [482.43, 484.36, 488.84, 495.15, 495.24, 502.69, 504.62,
+        ...         518.29, 519.1, 524.1, 524.12, 531.18, 548.42, 572.1,
+        ...         584.68, 609.09, 609.53, 666.63, 676.4]
+        >>> middle = [335.59, 338.43, 353.54, 404.27, 437.5, 469.01, 485.85,
+        ...          487.3, 493.08, 494.31, 499.1, 800]
+        >>> old = [519.01, 528.5, 530.23, 536.03, 538.56, 538.83, 557.24,
+        ...        558.61, 558.95, 565.43, 586.39, 594.69, 629.22, 645.69,
+        ...        691.84]
+        >>> A, p = alexandergovern_alt(young, middle, old)
+        >>> A, p
+        (6.941146076872535, 0.03109920451449096)
+    """
+
+    def calc_sem(data, mean, length):
+        """Calculates standard error using precalculated values"""
+        sum_ = np.sum((data - mean) ** 2)
+        d = length * (length - 1)
+        return (sum_ / d) ** .5
```

Comment:
```suggestion
        d = length * (length - 1)
        n = np.linalg.norm(data - mean)
        return n / d ** .5
```

Reply:
I believe this method was tested by Dominic and determined that it is slower than using ~`np.std / sqrt(len)`~.